dependencies.gradle

```diff
@@ -3,24 +3,35 @@ ext.versions = [
         name                : '0.1',
 
         minSdk              : 19,
-        targetSdk           : 24,
+        targetSdk           : 22,
```

Comment:
@atetc Robolectric does not support API level 24. Maybe change targetSdk on 19-23?
If i change support lib v '24.2.1' to '24.2.0' and TargetSDK to 22 Robolectric working fine. It have issue on google https://code.google.com/p/android/issues/detail?id=222597 and https://code.google.com/p/android/issues/detail?id=222597